includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -458,7 +485,7 @@ private function getFandomOverview() {
 					'brand' => 'games',
 					'title' => [
 						'type' => 'translatable-text',
-						'key' => 'global-footer-fandom-link-vertical-games'
+						'key' => 'global-footer-vertical-games'
```

Comment:
Please update to be compliant with https://github.com/Wikia/design-system-i18n/pull/15